redisson/src/main/java/org/redisson/RedissonMapCache.java

```diff
@@ -524,7 +524,7 @@ public V put(K key, V value, long ttl, TimeUnit unit) {
                 "        redis.call('zadd', lastAccessTimeSetName, currentTime, ARGV[2]);" +
                 "        local cacheSize = tonumber(redis.call('hlen', KEYS[1]));" +
                 "        if cacheSize > maxSize then" +
-                "            local lruItems = redis.call('zrange', lastAccessTimeSetName, 0, cacheSize - maxSize - 1);" +
+                "            local lruItems = redis.call('zrange', lastAccessTimeSetName, 0, -(maxSize+1));" +
```

Comment:
@johnou why do you use `-(maxSize+1)` instead ?

Reply:
looks like I misinterpreted the docs last night of how negative indexes behave https://redis.io/commands/zrange